### ISSUE TITLE
Page Header - Implement Up Button changes to Figma file

### ISSUE THREAD

**Issue by vavalos5** [grommet/hpe-design-system#2500]
As discussed with Eric and Scott, the following changes will be made to the Navigation Figma file:

- changing ascending name to up with an alias of back
- changing everything called ascending navigation to up navigation
- aligning ascending button examples to page content to align with page layout work
- utilizing anchor styling + including and keeping that previous icon

---------------------------------------------------------------------------------------------------------------
Navigation Page:
- Change Matrix to Cross Navigation

**Comment by vavalos5** [comment]
I’ve been working on making the Ascending Button for Navigation page easier to understand based off of the feedback we received from the user testing Chris Conetta conducted. We heard a common pattern on how “ascending or ascending button” didn’t resonate with them. So, in order to make it easier find this type of Navigation in our site and understand, I decided to call it Reverse Navigation. You’ll also notice i’m including browser back button in this section and changed ascending button for navigation to up navigation.

Here you can find the c[urrent page design](https://www.figma.com/file/b6hiokanRCwNnc93PpHdgQ/HPE-Design-System-Navigation?node-id=2862%3A20205).
[Option 1](https://www.figma.com/file/b6hiokanRCwNnc93PpHdgQ/HPE-Design-System-Navigation?node-id=3211%3A18102) I’m proposing
[Option 2](https://www.figma.com/file/b6hiokanRCwNnc93PpHdgQ/HPE-Design-System-Navigation?node-id=3212%3A18603) I’m proposing ( I like this example the best)

**Comment by vavalos5** [comment]
Received feedback from Kenny & Taylor. Awaiting for feedback from Scott for new design explorations found [here](https://www.figma.com/file/b6hiokanRCwNnc93PpHdgQ/HPE-Design-System-Navigation?node-id=3226%3A19631).

**Comment by vavalos5** [comment]
This ticket mentions how we were going to change the "Ascending button" to "Up Button". However, after many chats with Scott, we agreed on "In-App Reverse Navigation". I still think there should be an alias of "back" included as well. We've also included new diagrams to correctly represent the difference between the browser back button and the in-app reverse button.

I have also changed everything within the Navigation file that mentioned "ascending button" to "In-App Reverse Navigation" including the Navigation home page w/ cards. [Figma file can be found here.](https://www.figma.com/file/b6hiokanRCwNnc93PpHdgQ/HPE-Design-System-Navigation?node-id=3007%3A19071)
